documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -398,7 +398,22 @@ This setting is ignored and network install is disabled if `DISABLE_HDMI=1` is s
 
 In order to detect the keyboard, network install must initialise the USB controller and enumerate devices. This increases boot time by approximately 1 second so it may be advantageous to disable network install in some embedded applications.
 
-Default: `1` on Raspberry Pi 4 and Raspberry Pi 400, and `0` on Compute Module 4.
+Default: `1` on Raspberry Pi 4 and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.
```

Comment:
What does it default to on Pi 5?

Reply:
```suggestion
Default: `1` on Raspberry Pi 5, Raspberry Pi 4 and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.
```

Reply:
Presumably also `1` on Raspberry Pi 500 ? (since we're mentioning that anyway in this PR!)